### ISSUE TITLE
`torch.fx.node.normalized_arguments` does not pass `normalize_to_only_use_kwargs`

### ISSUE THREAD

**Issue by ylxdzsw** [pytorch/pytorch#62554]
## Issue description

`torch.fx.node.normalized_arguments` does not pass the `normalize_to_only_use_kwargs` argument to the `normalize_function` call.

## Code example

```
import torch
import torch.fx

class Tracer(torch.fx.Tracer):
    def is_leaf_module(self, m, module_qualified_name):
        return False

mod = torch.nn.Linear(4, 4)
net = Tracer().trace(mod)
node = list(net.nodes)[3]
node.normalized_arguments(mod)
# output: ArgsKwargsPair(args=(x, layers_0_weight), kwargs={'bias': layers_0_bias})
```

The expected output can be obtained by the following code:

```
from torch.fx.operator_schemas import normalize_function
normalize_function(node.target, node.args, node.kwargs, normalize_to_only_use_kwargs=True)
```

## System Info
PyTorch version: 1.9.0
Is debug build: False
CUDA used to build PyTorch: 10.2
ROCM used to build PyTorch: N/A

OS: [hide enterprise version] (x86_64)
GCC version: (GCC) 4.8.5 20150623 (Red Hat 4.8.5-39)
Clang version: Could not collect
CMake version: version 2.8.12.2
Libc version: glibc-2.17

Python version: 3.8.10 (default, Jun  4 2021, 15:09:15)  [GCC 7.5.0] (64-bit runtime)
Python platform: Linux-3.10.0-[hide enterprise version]-x86_64-with-glibc2.17
Is CUDA available: True
CUDA runtime version: 10.2.89
GPU models and configuration: 
GPU 0: Tesla V100-SXM2-16GB
GPU 1: Tesla V100-SXM2-16GB
GPU 2: Tesla V100-SXM2-16GB
GPU 3: Tesla V100-SXM2-16GB
GPU 4: Tesla V100-SXM2-16GB
GPU 5: Tesla V100-SXM2-16GB
GPU 6: Tesla V100-SXM2-16GB
GPU 7: Tesla V100-SXM2-16GB

Nvidia driver version: 440.95.01
cuDNN version: Could not collect
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] numpy==1.20.3
[pip3] torch==1.9.0
[pip3] torchaudio==0.9.0a0+33b2469
[pip3] torchvision==0.10.0
[conda] blas                      1.0                         mkl    defaults
[conda] cudatoolkit               10.2.89              hfd86e86_1    defaults
[conda] mkl                       2021.3.0           h06a4308_520    defaults
[conda] mkl-service               2.4.0            py38h7f8727e_0    defaults
[conda] mkl_fft                   1.3.0            py38h42c9631_2    defaults
[conda] mkl_random                1.2.2            py38h51133e4_0    defaults
[conda] numpy                     1.20.3           py38hf144106_0    defaults
[conda] numpy-base                1.20.3           py38h74d4b33_0    defaults
[conda] pytorch                   1.9.0           py3.8_cuda10.2_cudnn7.6.5_0    pytorch
[conda] torchaudio                0.9.0                      py38    pytorch
[conda] torchvision               0.10.0               py38_cu102    pytorch

**Comment by jamesr66a** [comment]
Thanks for the report! We will fix this
